lucetc/src/decls.rs

```diff
@@ -114,58 +114,58 @@ impl<'a> ModuleDecls<'a> {
         clif_module: &mut ClifModule<B>,
         bindings: &'a Bindings,
     ) -> Result<(), LucetcError> {
-        for ix in 0..decls.info.functions.len() {
-            let func_index = UniqueFuncIndex::new(ix);
+        // Get the name for this function from the module names section, if it exists.
```

Comment:
This is all pure code motion, no edits.

Reply:
an extremely cool trick to help review that it is in fact only indentation change: [?w=1](https://github.com/bytecodealliance/lucet/pull/391/files?w=1)